src/Sylius/Bundle/ChannelBundle/composer.json

```diff
@@ -41,6 +43,7 @@
         "psr-4": { "Sylius\\Bundle\\ChannelBundle\\spec\\": "spec/" }
     },
     "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
Ok, I opened RFC to add this to all bundles/components then - https://github.com/Sylius/Sylius/issues/4088.